ext/cache/sqlite.rs

```diff
@@ -285,29 +285,11 @@ impl Cache for SqliteBackedCache {
 async fn insert_cache_asset(
   db: Arc<Mutex<rusqlite::Connection>>,
   put: CachePutRequest,
-  body_key_start_time: Option<(String, u64)>,
+  response_body_key: Option<String>,
 ) -> Result<Option<String>, deno_core::anyhow::Error> {
   tokio::task::spawn_blocking(move || {
     let maybe_response_body = {
       let db = db.lock();
-      let mut response_body_key = None;
-      if let Some((body_key, start_time)) = body_key_start_time {
-        response_body_key = Some(body_key);
-          let last_inserted_at = db.query_row("
-          SELECT last_inserted_at FROM request_response_list
-          WHERE cache_id = ?1 AND request_url = ?2",
-          (put.cache_id, &put.request_url), |row| {
-            let last_inserted_at: i64 = row.get(0)?;
-            Ok(last_inserted_at)
-          }).optional()?;
-          if let Some(last_inserted) = last_inserted_at {
-            // Some other worker has already inserted this resource into the cache.
```

Comment:
This assumption isn't true as the response body can be different for multiple put operations with the same request.